app/repositories/meteors.repository.js

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+import { getApiUrl } from "../utils/getApiUrl.js";
+
+export const getMeteors = async (startDate, endDate) => {
+  const url = getApiUrl(startDate, endDate);
```

Comment:
you can form url right here

Reply:
Sorry, didn't catch your idea. Do you mean I shouldn't form url right here?

Reply:
yes